acceptance-test/src/acceptance-test/java/org/zalando/nakadi/webservice/BinaryEndToEndAT.java

```diff
@@ -91,4 +92,125 @@ public void testAvroPublishingAndJsonConsumption() throws IOException {
         Assert.assertEquals("1.0.0", metadata.get("version"));
         Assert.assertEquals(TEST_ET_NAME, metadata.get("event_type"));
     }
+
+    @Test
+    public void testAvroDeleteCannotWorkWhenCleanupPolicyIsDelete() throws IOException {
+        final var etName = "test-et-for-delete-cleanup-policy";
```

Comment:
maybe use postfix to create new event type for every run, that helps to avoid issues when consuming the event type